<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticlePanel</class>
 <widget class="QDockWidget" name="ParticlePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particles</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Edit Particle</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QComboBox" name="itemToAdd">
         <item>
          <property name="text">
           <string>(Particle 1)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Particle 2)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Particle 3)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Particle 4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Particle 5)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Polygon 1)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Polygon 2)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Polygon 3)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Polygon 4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>(Polygon 5)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="particleName">
         <property name="text">
          <string>pp</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLineEdit" name="particleFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>p2\p2</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Polygon Option:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="setButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Particle File:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Particle Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Item:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="delete_btn">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupDisplay">
        <property name="title">
         <string>Display Particle</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QCheckBox" name="display0">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Server</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Particle 1)</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Particle 2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Particle 3)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Particle 4)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Particle 5)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Polygon 1)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Polygon 2)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Polygon 3)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Polygon 4)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="display10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>(Polygon 5)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupKeepInitial">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Keep Initial Step</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QCheckBox" name="keepInitial0">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepInitial10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupKeepFinal">
        <property name="title">
         <string>Keep Final Step</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QCheckBox" name="keepFinal0">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="keepFinal10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
